content/manuals/docker-hub/_index.md

```diff
@@ -32,7 +32,7 @@ Docker Hub is a service provided by Docker for finding and sharing container ima
 
 It's the world’s largest repository of container images with an array of content sources including container community developers, open source projects, and independent software vendors (ISV) building and distributing their code in containers.
 
-Docker Hub is also where you can go to [change your Docker account settings and carry out administrative tasks](/admin/). If you have a Docker Team or Business subscription, you can also carry out administrative tasks in the [Docker Admin Console](https://admin.docker.com).
+Docker Hub is also where you can go to [carry out administrative tasks for organizations](/admin/). If you have a Docker Team or Business subscription, you can also carry out administrative tasks in the [Docker Admin Console](https://admin.docker.com).
```

Comment:
Just a question, not a blocker: is the Docker Admin Console URL `https://admin.docker.com` or `app.docker.com/admin`?

Reply:
Good catch. It should be `app.docker.com/admin` but the old URL, `admin.docker.com` will redirect to it. Capturing this as a follow-up in ENGDOCS-2246 to get this out asap and because there's probably more than one wrong occurrence.